src/androidTest/java/com/couchbase/lite/mockserver/MockDocumentBulkGet.java

```diff
@@ -72,7 +71,15 @@ public MockResponse generateMockResponse(RecordedRequest request) {
 
         try {
 
-            Map <String, Object> bulkDocsJson = Manager.getObjectMapper().readValue(request.getUtf8Body(), Map.class);
+            byte[] body = request.getBody();
+
+            // for gzip support core java #172
+            // https://github.com/couchbase/couchbase-lite-java-core/issues/172
```

Comment:
I think these comments aren't needed and can be removed, especially if we wrap this in a call like `MockHelper.isGzip(request)`